asapdiscovery-data/asapdiscovery/data/schema_v2/ligand.py

```diff
@@ -112,12 +73,12 @@ class Ligand(DataModelAbstractBase):
     tags: dict[str, str] = Field({}, description="Dictionary of SD tags")
 
     data: str = Field(
-        "",
+        ...,
         description="SDF file stored as a string to hold internal data state",
         repr=False,
     )
-    data_format: DataStorageType = Field(
-        DataStorageType.sdf,
+    data_format: Literal["sdf"] = Field(
```

Comment:
Can we keep the enum? It should cast easily to a string anyway, but we can use `Literal`

Reply:
Found a way to do it just had to run it through JSON to get it to work!